raiden/tests/utils/smartcontracts.py

```diff
@@ -109,7 +109,8 @@ def deploy_service_registry_and_set_urls(
     )
 
     # Test that getting a random service for an empty registry returns None
-    assert get_random_service(c1_service_proxy) is None
+    pfs_address, pfs_eth_address = get_random_service(c1_service_proxy)
+    assert pfs_address is None and pfs_eth_address is None
```

Comment:
nitpick: Both here and in other tests, it would be better to have one assertion at each line. The reasoning is that if and when they fail you will get an error message showing you exactly what failed. While if you combine it with `and` you won't know what went wrong.